src/pages/LoginPage/LoginPage.jsx

```diff
@@ -33,23 +37,53 @@ const LoginPage = () => {
           className='input-box'
           id='password'
           value={loginUser.password}
-          onChange={(e) =>
-            setLoginUser((prev) => ({ ...prev, password: e.target.value }))
-          }
+          onChange={(e) => {
+            setError("noerror");
+            setLoginUser((prev) => ({ ...prev, password: e.target.value }));
+          }}
+          onFocus={() => setError("noerror")}
+          required
         />
         <div className='forgot-psswrd'>
           <label>
-            <input type='checkbox' name='checkbox' className='checkbox-size' />
+            <input
+              type='checkbox'
+              name='checkbox'
+              className='checkbox-size'
+              defaultChecked
+            />
             Remember Me
           </label>
-          <p className='red-color forgot-psswrd-margin'>Forgot Password?</p>
+
+          <p
+            onClick={() => {
+              setLoginUser((prev) => ({
+                ...prev,
+                email: "adarshbalika@gmail.com",
```

Comment:
you can store guest login credentials in a separate file like this
```
export const GUEST_CREDENTIAL = {  email: "adarshbalika@gmail.com"
password: "adarshbalika"  } 

```
and then use the variable